packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1722,6 +1722,17 @@ const schema: SchemaType<DbUser> = {
     label: "Hide the frontpage book ad"
   },
 
+  hideFrontpageBook2020Ad: {
+    type: Boolean,
+    canRead: [userOwns, 'sunshineRegiment', 'admins'],
+    canCreate: ['members'],
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
+    optional: true,
```

Comment:
should also be `nullable: true`, if we're not giving it a `schemaDefaultValue`.  (giving it a `schemaDefaultValue` would make the schema migration which creates the column in the database also assign a value for all existing users, so there'd be no need to make it nullable then.)